src/configs/getAndroidConfig.ts

```diff
@@ -15,10 +15,16 @@ limitations under the License.
 */
 
 import { Platform } from "style-dictionary/types/Platform";
+import { TransformedToken } from "style-dictionary/types";
 import { Theme } from "../@types";
+import { isCoreColor, isNotCoreColor } from "../filters/isCoreColor";
 import _ from "lodash";
 
-export default function getAndroidConfig(theme: Theme): Platform {
+function isFontType(token: TransformedToken) {
+  return ['typography', 'fontFamilies', 'fontWeights', 'letterSpacing', 'fontSizes', 'lineHeights'].includes(token.type);
+}
```

Comment:
That probably ought to be in a helper file, as this will be re-used across all platforms.
Could you add a JSDoc comment around the funciton too please?

Reply:
I created a separate filter in 6c800e700355ed498ef0928e10562c44372d0e96.